### PR TITLE
fix: auto focus and select the input when notification modal prompt

### PR DESCRIPTION
**Pull request by akumatus**
- Use `autofill` attribute instead of `input.focus()` on ref
- Auto focus and select all content of the input when prompt

### CODE REVIEW

**Comment by akumatus** [comment]
* **#7286** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7286?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @akumatus and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by JimmFly** [comment]
This asynchronous autofocus function is used to handle some situations where autofocus cannot be achieved, such as renaming the collection at sidebar, creating a workspace, deleting a workspace, etc. It would be great if there were no problems after the change.
image

**Review by Saul-Mirone — APPROVED** [review]

**Review by EYHN — APPROVED** [review]

File: packages/frontend/component/src/ui/input/input.tsx [hunk 1]
```diff
@@ -82,11 +68,12 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
     >
       {preFix}
       <input
+        autoFocus={autoFocus}
         className={clsx(input, {
           large: size === 'large',
           'extra-large': size === 'extraLarge',
         })}
-        ref={autoFocus ? handleAutoFocus : upstreamRef}
```
Comment (EYHN) [inline packages/frontend/component/src/ui/input/input.tsx]: Can you check other places where this component is used is work fine?

Comment (akumatus) [reply packages/frontend/component/src/ui/input/input.tsx]: It seems something was wrong after I modified.This part of the code has been reset.

**Pull request merged**